### DISCUSSION TITLE
Datatype properties

### DISCUSSION THREAD

**Discussion by ManuelSimoes** [nemo-ufes/gufo#31] [Q&A] [answered]
Can we interpret all datatype properties, minus the functional properties (hasBeginPointInXSDDate hasBeginPointInXSDDateTimeStamp hasEndPointInXSDDate hasEndPointInXSDDateTimeStamp), as subproperties of concernsQualityValue, hasQualityValue or hasValueComponent?

**Comment by jpalmeida** [comment, marked as answer]
I'm not sure I understand the question. Do you mean that all datatype properties in gUFO-based ontologies should be declared as specializations of those? There's no such recommendation.

**Reply by ManuelSimoes** [reply]
That was my question indeed, as those 3 properties are listed as the only datatype properties.  
Thanks for clarifying!
